src/base/scanfoldersmodel.cpp

```diff
@@ -341,7 +341,7 @@ void ScanFoldersModel::configure()
 {
     const QVariantHash dirs = Preferences::instance()->getScanDirs();
 
-    for (QVariantHash::const_iterator i = dirs.begin(), e = dirs.end(); i != e; ++i) {
+    for (auto i = dirs.begin(), e = dirs.end(); i != e; ++i) {
```

Comment:
why not:
`for (auto i = dirs.begin(); i != dirs.end(); ++i) {`

or:
`for (auto i = dirs.cbegin(); i != dirs.cend(); ++i) {`

Reply:
Changed.